CMakeLists.txt

```diff
@@ -51,6 +51,9 @@ add_library(polaris_cpp_client
 if (MSVC)
     target_compile_definitions(polaris_cpp_client PRIVATE BUILDING_DLL)
 endif()
+if (POLARIS_ENABLE_TLS)
```

Comment:
Does this get enabled by default in cmake? I just would like to make it consistent with bazel where secure is the default.

Reply:
Yes, it's inherited from the C library's file, and it's set to `ON` by default in there.